Core/SiteRating.swift

```diff
@@ -145,11 +145,11 @@ public class SiteRating {
     }
 
     private func uniqueMajorTrackerNetworks(trackers: [DetectedTracker: Int]) -> Int {
-        return Set(trackers.keys.filter({ majorTrackerNetworkStore.network(forName: $0.networkName ?? "" ) != nil }).flatMap({ $0.networkName })).count
+        return Set(trackers.keys.filter({ majorTrackerNetworkStore.network(forName: $0.networkName ?? "" ) != nil }).compactMap({ $0.networkName })).count
```

Comment:
What is this change for?

Reply:
`flatMap` has been deprecated, so this is now using the correct API